File: factorial.py
def factorial(num): #create a function
  fact = 1 #initialize factorial as 1
  for i in range(1,num+1): #loop to the number
      fact *=  i #keep multiplying untill the number is reached
  print (f"The factorial of {num} is: {fact}") #print the result

num = int(input("Enter a number: ")) #alternatively you could pass in an integer
factorial(num) #call the function

File: c-cpp/factorial.cpp
#include <iostream>

void factorial(int n) {
  int fact = 1;
  for (int i = 1; i <= n; i++) {
    fact *= i;
  }
  std::cout << "The factorial of " << n << " is: " << fact;
}

int main() {
  factorial(5);
}
